core/testdata/testspecs/v2_specs.go

```diff
@@ -588,8 +588,15 @@ func GenerateBlockhashStoreSpec(params BlockhashStoreSpecParams) BlockhashStoreS
 		params.RunTimeout = 15 * time.Second
 	}
 
-	if params.FromAdress == "" {
-		params.FromAdress = "0x4bd43cb108Bc3742e484f47E69EBfa378cb6278B"
+	var formattedFromAddresses string
+	if params.FromAddresses == nil {
+		formattedFromAddresses = "[\"0x4bd43cb108Bc3742e484f47E69EBfa378cb6278B\"]"
```

Comment:
nit/ could avoid the escaping
```suggestion
		formattedFromAddresses = `["0x4bd43cb108Bc3742e484f47E69EBfa378cb6278B"]`
```